### DISCUSSION TITLE
Folder location of Exentsion

### DISCUSSION THREAD

**Discussion by thannerfabian** [coder/code-server#5230] [Q&A] [answered]
Hello,
I need to change a JSON File on one of my installed Extensions. I installed it from VSIX.
Where can i find this folder? On a normal VS-Code installation it´s $HOME/.vscode
Thanks in advance

**Comment by jsjoeio** [comment, marked as answer]
Should be under: `$HOME/.local/share/code-server/extensions`

**Reply by thannerfabian** [reply]
Thank you very much!
